docs/getting-started.md

```diff
@@ -3,44 +3,9 @@ title: Getting Started
 category: 5d4c940cae4e610060475769
 ---
 
-<p align="center">
-  <img width="400" src="https://raw.githubusercontent.com/readmeio/api/main/docs/images/logo.svg" />
-</p>
+<!--
```

Comment:
I wonder what rdmd does with comments in Markdown.

Reply:
It just doesn't render them out as HTML comments or anything at all, which is actually quite handy for internal dev notes like this. Take [this page](https://docs.readme.com/docs/rdme) for example. Here's some comments from its Markdown source: https://github.com/readmeio/rdme/blame/main/documentation/rdme.md#L7-L18

Reply:
oh sweet